src/components/BlockingViews/FullPageNotFoundView.js

```diff
@@ -24,11 +28,21 @@ const defaultProps = {
 const FullPageNotFoundView = (props) => {
     if (props.shouldShow) {
         return (
-            <BlockingView
-                icon={Expensicons.QuestionMark}
-                title={props.translate('notFound.notHere')}
-                subtitle={props.translate('notFound.pageNotFound')}
-            />
+            <View style={[styles.flex1, styles.dFlex, styles.flexColumn]}>
```

Comment:
Just wondering if the wrapping View with flex is necessary.

```js
              <>
                <HeaderWithCloseButton
                    shouldShowBackButton
                    onBackButtonPress={() => Navigation.dismissModal()}
                    onCloseButtonPress={() => Navigation.dismissModal()}
                />
                <BlockingView
                    icon={Expensicons.QuestionMark}
                    title={props.translate('notFound.notHere')}
                    subtitle={props.translate('notFound.pageNotFound')}
                />
            </>
```

seems to give the same results in web, did you see a case where this doesn't work?

Reply:
I tried a plain `View` and the layout didn't work but looks like fragment does for our use case. I'll update it 👍🏽